gdcdictionary/schemas/exposure.yaml

```diff
@@ -291,6 +328,146 @@ properties:
         $ref:
           - "_terms_enum.yaml#/tobacco/common"
 
+  occupation_duration_years:
+    type: integer
+    maximum: 2050
+    minimum: 1900
+
+  occupation_type:
+    type: array
+    items:
+      enum:
+        - Legislators and Senior Officials
+        - Managing Directors and Chief Executives
```

Comment:
the enums added are not the same as the ones in the jira. I'm guessing there's a separate list somewhere?

Reply:
updated JIRA, enums in github are correct (level 3 groupings from International Standard Classification of Occupations types); also @wwysoc2 the list in JIRA is alphabetized if that will help you

Reply:
Alphabetized 👍